ajax/libs/jquery.x18n/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "jquery.x18n",
+  "version": "1.1.3",
+  "description": "A JavaScript internationalisation library",
```

Comment:
Let's use this description https://github.com/florian/jQuery.x18n/blob/master/jQuery.x18n.jquery.json#L4
This description is mentioned `x18n` part, which I think is good than current description.
What do you think?

Thanks.